### PR TITLE
Fixes for #224

### PR DESCRIPTION
**Pull request by philkra**
code snippets and more for https://github.com/xataio/mdx-docs/pull/224

### CODE REVIEW

**Review by joan-ing — APPROVED** [review]
lgtm

File: 040-SDK/130-transaction.mdx [hunk 1]
```diff
@@ -266,50 +266,58 @@ const result = await xata.transactions.run([
...
-  {
-    "update": {
-      "table": "Users",
-      "id": "rec_cmf572veoo9ahn3r96p0",
-      # Include the fields to update for this record
-    }
-  },
-  # Additional transactions can be added here
-])
+result = xata.records().transactions({
```
Comment (kostasb) [inline 040-SDK/130-transaction.mdx:274]: @philkra should be `transaction` not transactions right?

`xata.records().transaction` from https://github.com/xataio/xata-py/blob/main/xata/api/records.py#L34

Comment (philkra) [reply 040-SDK/130-transaction.mdx:274]: Oh yikes! yes it should, let me open a PR

Comment (joan-ing) [reply 040-SDK/130-transaction.mdx:274]: if this is not already open, I'll just commit it in the main PR. I have to push other changes as well

Comment (philkra) [reply 040-SDK/130-transaction.mdx:274]: https://github.com/xataio/mdx-docs/pull/226

**Pull request merged**